cmake/pcl_targets.cmake

```diff
@@ -906,6 +906,16 @@ macro(PCL_SET_TEST_DEPENDENCIES _var _module)
   set(${_var} global_tests ${_module} ${PCL_SUBSYS_DEPS_${_module}})
 endmacro()
 
+###############################################################################
+# Add two test targets for both values of PCL_RUN_TESTS_AT_COMPILE_TIME 
```

Comment:
Might be better to have the macro name in CMake different from the macro defined for compilation. Easier to mention in docs

Reply:
not sure i got the rationale. could you please elaborate on that?